go.mod

```diff
@@ -53,11 +53,11 @@ require (
 	go.uber.org/thriftrw v1.29.2
 	go.uber.org/yarpc v1.70.3
 	go.uber.org/zap v1.13.0
-	golang.org/x/net v0.17.0
+	golang.org/x/net v0.19.0
```

Comment:
should this be in a separate PR?

Reply:
That comes from
```
go get golang.org/x/exp
```
Not sure how to separate them apart. All x/ packages are linked to one another.

Reply:
ah, ok I didn't see `exp` being added

Reply:
yes. It has a nice helper of maps.Keys